### ISSUE TITLE
Failed to list profile on EM05-CE

### ISSUE THREAD

**Issue by ImproperCatGirl** [stich86/lpac-libmbim-wrapper#14]
My configuration:
    wwan card : EM05-CE (works with Windows LPA)
    eSIM chip (since there is no integrated esim onboard the EM05-CE)  : A G+M chip salvaged from iPhone (tested usable with LPAC PCSC mode and Android OpenEUICC)
The error:
```
sudo lpac-mbim --device=/dev/cdc-wdm0 profile list
INFO: Connect
INFO: Open channel with AID a0000005591010ffffffff8900000100
Error executing command: error: operation failed: Failure

Traceback (most recent call last):
  File "/usr/local/bin/lpac-mbim", line 164, in <module>
    main()
  File "/usr/local/bin/lpac-mbim", line 149, in main
    payload = handle_type_apdu(req["payload"]["func"], req["payload"]["param"])
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/bin/lpac-mbim", line 75, in handle_type_apdu
    channel_id = open_channel(param)
                 ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/bin/lpac-mbim", line 51, in open_channel
    output_cmd = run_mbimcli(f'--ms-set-uicc-open-channel="application-id={aid}",channel-group=1').split('\n')
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'split'
```

mmcli modem info:
```
  --------------------------------
  General  |                 path: /org/freedesktop/ModemManager1/Modem/0
           |            device id: 
  --------------------------------
  Hardware |         manufacturer: Quectel
           |                model: Quectel EM05-CE
           |    firmware revision: EM05CEFCR08A07M1G_LNV
           |         h/w revision: QUECTEL Mobile Broadband Modul
           |            supported: gsm-umts, lte
           |              current: gsm-umts, lte
           |         equipment id: 
  --------------------------------
  System   |               device: /sys/devices/pci0000:00/0000:00:14.0/usb3/3-2
           |              drivers: option, cdc_mbim
           |               plugin: quectel
           |         primary port: cdc-wdm0
           |                ports: cdc-wdm0 (mbim), ttyUSB0 (at), wwp0s20f0u2i1 (net)
  --------------------------------
  Status   |                 lock: sim-pin2
           |       unlock retries: sim-pin2 (3)
           |                state: connected
           |          power state: on
           |          access tech: lte
           |       signal quality: 29% (recent)
  --------------------------------
  Modes    |            supported: allowed: 2g, 3g, 4g; preferred: none
           |              current: allowed: 2g, 3g, 4g; preferred: none
  --------------------------------
  IP       |            supported: ipv4, ipv6, ipv4v6
  --------------------------------
  3GPP     |                 imei: 
           |        enabled locks: fixed-dialing
           |          operator id: 46011
           |        operator name: CHN-CT
           |         registration: roaming
           | packet service state: attached
  --------------------------------
  3GPP EPS | ue mode of operation: ps-2
  --------------------------------
  SIM      |     primary sim path: /org/freedesktop/ModemManager1/SIM/0
  --------------------------------
  Bearer   |                paths: /org/freedesktop/ModemManager1/Bearer/0
```

**Comment by vmsh0** [comment]
Hello,
I have an EM05-G in my Lenovo laptop and I was also getting the same issue.

I have solved the issue by switching to the second SIM slot, which in the case of my system (not sure if it's true in general) it's apparently the slot connected to the eSIM: `mbimcli -p -d /dev/cdc-wdm0 --ms-set-device-slot-mappings=1`

After this, I have successfully enumerated the profiles on the eSIM. I have not attempted anything further.
